### PR TITLE
Add a lint for paths ending .worker.html

### PR DESCRIPTION
**Pull request by jgraham**
This change is [Reviewable](https://reviewable.io/reviews/w3c/wpt-tools/141)

### CODE REVIEW

**Comment by gsnedders** [comment]
Please also add lints for `.any.js` turning into `.any.html` and `.any.worker.html`.

**Comment by gsnedders** [comment]
plz fix the commit message to match what the commit actually does now, but with that fixed it LGTM

**Comment by Ms2ger** [comment]
Tests please.

File: lint/lint.py [hunk 1]
```diff
@@ -63,6 +63,18 @@ def check_path_length(repo_root, path):
     return []
 
 
+def check_worker_collision(repo_root, path):
+    endings = [(".any.html", ".any.js"),
+               (".any.worker.html", ".any.js"),
+               (".worker.html", ".worker.js")]
+    for path_ending, generated in endings:
+        if path.endswith(path_ending):
+            return [("WORKER COLLISION", "path ends with %s which collides with generated tests from %s files" % (path_ending, generated),
```
Comment (Ms2ger) [inline lint/lint.py]: One more nit: please put each of the tuple fields on its own line, rather than having the first two together here.

Comment (jgraham) [inline lint/lint.py]: I think you misread that? The tuple after the `%` is just filling in values in the string.

Comment (Ms2ger) [inline lint/lint.py]: First is `"WORKER COLLISION"`, second is `"path ends …generated)`

**Pull request merged**